addons/im_livechat_mail_bot/models/res_users.py

```diff
@@ -1,12 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import models, fields
-
-
-class Users(models.Model):
-    _inherit = 'res.users'
-
-    odoobot_state = fields.Selection(selection_add=[
-        ('onboarding_canned', 'Onboarding canned'),
-    ], ondelete={'onboarding_canned': lambda users: users.write({'odoobot_state': 'disabled'})})
```

Comment:
In regard to upgrade script, isn't it going to write `disabled` rather than keeping `onboarding_canned` when deleting the module?

Reply:
I asked upgrade team and changed the upgrade script to `merge_module`. `ondelete` won't be triggered here.